keyboards/akko/5075/info.json

```diff
@@ -0,0 +1,271 @@
+                {"label": "&", "matrix": [1, 7], "x": 7, "y": 1.25},
+                {"label": "*", "matrix": [1, 8], "x": 8, "y": 1.25},
+                {"label": "(", "matrix": [1, 9], "x": 9, "y": 1.25},
+                {"label": ")", "matrix": [1, 10], "x": 10, "y": 1.25},
+                {"label": "_", "matrix": [1, 11], "x": 11, "y": 1.25},
+                {"label": "+", "matrix": [1, 12], "x": 12, "y": 1.25},
+                {"label": "Bksp", "matrix": [1, 13], "x": 13, "y": 1.25, "w": 2},
+                {"label": "Home", "matrix": [1, 15], "x": 15.5, "y": 1.25},
+                {"label": "Tab", "matrix": [2, 0], "x": 0, "y": 2.25, "w": 1.5},
+                {"label": "Q", "matrix": [2, 1], "x": 1.5, "y": 2.25},
+                {"label": "W", "matrix": [2, 2], "x": 2.5, "y": 2.25},
+                {"label": "E", "matrix": [2, 3], "x": 3.5, "y": 2.25},
+                {"label": "R", "matrix": [2, 4], "x": 4.5, "y": 2.25},
+                {"label": "T", "matrix": [2, 5], "x": 5.5, "y": 2.25},
+                {"label": "Y", "matrix": [2, 6], "x": 6.5, "y": 2.25},
+                {"label": "U", "matrix": [2, 7], "x": 7.5, "y": 2.25},
+                {"label": "I", "matrix": [2, 8], "x": 8.5, "y": 2.25},
+                {"label": "O", "matrix": [2, 9], "x": 9.5, "y": 2.25},
+                {"label": "P", "matrix": [2, 10], "x": 10.5, "y": 2.25},
+                {"label": "{", "matrix": [2, 11], "x": 11.5, "y": 2.25},
+                {"label": "}", "matrix": [2, 12], "x": 12.5, "y": 2.25},
+                {"label": "|", "matrix": [2, 13], "x": 13.5, "y": 2.25, "w": 1.5},
+                {"label": "PageUp", "matrix": [2, 15], "x": 15.5, "y": 2.25},
+                {"label": "Caps Lock", "matrix": [3, 0], "x": 0, "y": 3.25, "w": 1.75},
+                {"label": "A", "matrix": [3, 1], "x": 1.75, "y": 3.25},
+                {"label": "S", "matrix": [3, 2], "x": 2.75, "y": 3.25},
+                {"label": "D", "matrix": [3, 3], "x": 3.75, "y": 3.25},
+                {"label": "F", "matrix": [3, 4], "x": 4.75, "y": 3.25},
+                {"label": "G", "matrix": [3, 5], "x": 5.75, "y": 3.25},
+                {"label": "H", "matrix": [3, 6], "x": 6.75, "y": 3.25},
+                {"label": "J", "matrix": [3, 7], "x": 7.75, "y": 3.25},
+                {"label": "K", "matrix": [3, 8], "x": 8.75, "y": 3.25},
+                {"label": "L", "matrix": [3, 9], "x": 9.75, "y": 3.25},
+                {"label": ":", "matrix": [3, 10], "x": 10.75, "y": 3.25},
+                {"label": "\"", "matrix": [3, 11], "x": 11.75, "y": 3.25},
+                {"label": ":", "matrix": [3, 12], "x": 12.75, "y": 3.25},
+                {"label": "Enter", "matrix": [3, 13], "x": 13.75, "y": 3.25, "w": 1.25},
+                {"label": "PageDn", "matrix": [3, 15], "x": 15.5, "y": 3.25},
+                {"label": "Shift", "matrix": [4, 0], "x": 0, "y": 4.25, "w": 1.25},
+                {"label": ":", "matrix": [4, 11], "x": 1.25, "y": 4.25},
```

Comment:
Correct me if I'm wrong, but I belive this is an ANSI board, so this key doesn't exist (at least i could not find an ISO version for the 5075, only 5075S).